README.md

```diff
@@ -1,268 +1,239 @@
-  - Merge your commit into master. If you are following the fork and pull request workflow, this will involve opening a PR, possibly getting approval, and then merging.
+Each folder contains detections in the format of `<log/resource type>_<detecton_type>`:
 
-#### Updating
+* **Rules** analyze [logs](https://docs.runpanther.io/log-analysis/supported-logs) to detect malicious activity
+* **Policies** represent the desired secure state of a [resource](https://docs.runpanther.io/cloud-security/resources) to detect security misconfigurations
+* **Scheduled rules** (coming soon) analyze output of periodically executed [SQL queries](https://docs.runpanther.io/enterprise/data-analytics/example-queries)
 
-Now that you have a private repository will all the default policies and rules, you can customize away to your hearts content. When you are ready pull in the newest changes from our public repository, perform the following steps from within your private repo:
+Global helper functions are defined in the `global_helpers` folder. This is a hard coded location and cannot change. However, you may create as many files as you'd like under this path. Simply import them into your detections by the specified `GlobalID`.
 
-  - If you have not already done so, add the public repository as a remote: `git remote add panther-upstream git@github.com:panther-labs/panther-analysis.git`
-  - Pull in the latest changes (in a local branch if you cannot push to master): `git pull panther-upstream master`
-    - You may need to use the `--allow-unrelated-histories` flag if you did not maintain the history originally
-  - Push the latest changes up and merge them: `git push`
+Additionally, groups of detections may be linked to multiple "Reports", which is a system for tracking frameworks like CIS, PCI, MITRE ATT&CK, or more.
 
-## Panther Analysis Tool
+# Usage
 
-The `panther_analysis_tool` is a Python command line interface for testing, packaging, and deploying Panther Policies and Rules. Consider checking it out here [panther\_analysis\_tool](https://github.com/panther-labs/panther_analysis_tool) for help managing your Panther analysis packs.
+*For a full reference on writing detections, read our [docs](https://docs.runpanther.io/user-guide/analysis/panther-analysis-tool)!*
 
-## Writing Policies
+Each detection has a Python file (`.py`) and a metadata file (`.yml`) of the same name (in the same location), for example:
 
-Each Panther Policy consists of a Python body and a YAML or JSON specification file.
+`okta_brute_force_logins.py`
 
-In the Python body, returning a value of `True` indicates the resource being evaluated is compliant. Returning a value of `False` indicates the resource is non-compliant, and an alert may be sent or an auto-remediation may be performed as a result.
+```python
+def rule(event):
+    return (event['outcome']['result'] == 'FAILURE' and
+            event['eventType'] == 'user.session.start')
```

Comment:
Should we advertise/use `.get()` in the example, as a 'best practice'? 

Should we also list some 'best practices when writing detections' (or maybe in the full doc)?

Reply:
I'd say we should put best practices into our docs

Reply:
And outcome -> result is always present in the log too I believe